README.md

```diff
@@ -12,24 +12,101 @@ Protein-coding annotation extension using additional homology searches against l
 
 The HyPro tool extends common protein-coding annotations made with [Prokka](https://github.com/tseemann/prokka) using additional homology searches. The approach currently takes a gff input file, extracts the sequences of hypothetical proteins and searches against a selected database (available are [UniProtKB](ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/), [Uniref50](ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref50/), [Uniref90](ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref90/), [Unref100](ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref90/) and [Protein DB](ftp://ftp.wwpdb.org/pub/pdb/derived_data/)) to find homologs. For searching, [MMseqs2](https://github.com/soedinglab/MMseqs2) is utilized which offers a fast and accurate sequence comparison.
 
-The tool has been tested in a conda environment (v4.7.11).
+The tool shas been tested on a conda (v4.10.1) and docker engine (v20.10.7)
 
 ## Tool Composition:
 
-- **hypro.py**     main script to be called by the user
-- **mmseqs2.sh**     bash script comprising all required mmseqs2 commands (createdb, createindex, search) and output formatting
+- **main.nf** : main script to be called by the user
+- **prokka_annotation.nf**  : process for _de novo_ annotation of input fasta
+- **mmseqs2.nf**  : process for running additional annotation of input fasta using input query DB
+- **update_prokka.nf**  : process for extending prokka annotation by annotatinos found during MMseqs2 process
 
 ## Requirements
-HyPro requires the provided list of software to function properly. 
+HyPro requires the provided list of software to function properly.
 
 |Program/Package|Version|Note|
 |---------------|-------|------|
+|nextflow|21.04.1|Might also work for other nextflow versions|
+|docker|20.10.7|Used for running HyPro on docker engine. Might also work for other docker versions|
 |python|3.7|Might also work for other python3 versions|
-|mygene|3.1.0|Might also work for other versions|
-|pandas|0.25.2|Might also work for other versions.|
-|mmseqs2|10.6d92c|Install in conda environment|
-|prokka (recommended)|1.14.6|used for _de novo_ annotation of test data of chlamydia|
+|pandas|0.25.2|Might also work for other versions|
+|mygene|3.1.0|Automatically installed when running HyPro on a conda or docker engine. Might also work for other versions.|
```

Comment:
Basically w/ the current implementation people only need nextflow & docker. We can still mention and honor the other stuff but currently it looks like the user needs to install this

Reply:
ahh, I just see that you added this is a Note. Okay, maybe we just add another sentence that only nf and docker or conda need to be installed?